odoo/content.md

```diff
@@ -70,6 +70,92 @@ $ docker run -p 8071:8069 --name odoo3 --link db:db -t odoo
 
 Please note that for plain use of mails and reports functionalities, when the host and container ports differ (e.g. 8070 and 8069), one has to set, in Odoo, Settings->Parameters->System Parameters (requires technical features), web.base.url to the container port (e.g. 127.0.0.1:8069).
 
+## Use with Docker Compose
+
+The Odoo Docker image has some environment variables (with default values) that you can set to fit your needs:
+
+-	`HOST`: the name of the "host" or address of the PostgreSQL database inside the Odoo container. By default it's: `db`.
+-	`PORT`: the port in where that database container is listening. By default it's the standard PostgreSQL port: `5432`.
```

Comment:
"The port the PostgreSQL Server is listening to. By default, it's the standard PostgreSQL port:..."